BQ25895M.device.lib.nut

```diff
@@ -291,11 +241,88 @@ class BQ25895M {
     
     //-------------------- PRIVATE METHODS --------------------//
     
+    // Set the enable charging bit, charging will happen automatically
+    function _enableCharging() {
```

Comment:
I don't think this method is needed anymore - see comments in enableCharger mehtod